internal/utils/files/files.go

```diff
@@ -8,17 +8,44 @@ import (
 	testutils "github.com/kudobuilder/kuttl/internal/utils"
 )
 
+var defaultIgnorePatterns = []string{"README*"}
+
+// matchesAnyPattern checks if a filename matches any of the given patterns.
+// Returns true if a match is found, false otherwise.
+func matchesAnyPattern(filename string, patterns []string, logger testutils.Logger) bool {
+	for _, pattern := range patterns {
+		matched, err := filepath.Match(pattern, filename)
+		if err != nil {
+			logger.Logf("Invalid ignore pattern %q: %v", pattern, err)
```

Comment:
Looks like this will be super spammy. Could we (probably additionally) detect bad patterns early (when reading the config?) and fail there instead?